tests/openvino/test_exporters_cli.py

```diff
@@ -263,6 +263,42 @@ class OVCLIExportTestCase(unittest.TestCase):
                 {"f8e5m2": 2, "int4": 28},
             ],
         ),
+        (
```

Comment:
Could you please also add these tests to `test_quantization.py::OVQuantizerTest.test_ov_model_static_quantization_with_auto_dataset` ?